### PR TITLE
Images in more-ons and story packages

### PR DESCRIPTION
**Pull request by georgecrawford**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,9 +23,11 @@
     "bower": "^1.7.7",
     "cheerio": "^0.19.0",
     "cookie-parser": "^1.4.1",
+    "denodeify": "^1.2.1",
```
Comment (quarterto) [inline package.json]: I've been [using](https://github.com/Financial-Times/google-amp/blob/532b541f4ef26dd96ec3376abd28e30f2ff0d78a/server/lib/compile-scss.js#L3) [`@quarterto/promisify`](https://gist.github.com/quarterto/726a8947687cb33ca907). Don't mind which we go for as long as it's consistent

Comment (georgecrawford) [inline package.json]: Thanks, fixed.

**Comment by quarterto** [comment]
@georgecrawford can you rebase this on master please?

**Comment by georgecrawford** [comment]
Doing so now :)




George

___________________

George Crawford
Senior Developer, Financial Times

+44 (0)752 567 9717
Number One Southwark Bridge,
London SE1 9HL



> On 24 Feb 2016, at 15:23, Matt Brennan <notifications@github.com> wrote:
> 
> @georgecrawford can you rebase this on master please?
> 
> —
> Reply to this email directly or view it on GitHub.
> 


-- 

------------------------------

*This email was sent by a company owned by Financial Times Group Limited 
("FT Group <http://aboutus.ft.com/corporate-information/#axzz3rajCSIAt>"), 
registered office at Number One Southwark Bridge, London SE1 9HL.  
Registered in England and Wales with company number 879531. This e-mail may 
contain confidential information. If you are not the intended recipient, 
please notify the sender immediately, delete all copies and do not 
distribute it further.  It could also contain personal views which are not 
necessarily those of the FT Group.  We may monitor outgoing or 
incoming emails as permitted by law.*

**Comment by quarterto** [comment]
Looking good!

**Pull request merged**